frontend/src/actions/cards.js

```diff
@@ -40,6 +41,21 @@ export function fetchCardDetail(cardId) {
   };
 }
 
+export function updateUserCards(cardId) {
```

Comment:
Lets name this something related to transactions, as it is not updating the users existing cards. Maybe `putTransaction`?